src/notification/toast-notification.component.js

```diff
@@ -190,7 +190,7 @@ angular.module( 'patternfly.notification' ).component('pfToastNotification', {
     'data': '=?'
   },
   templateUrl: 'notification/toast-notification.html',
-  controller: function () {
+  controller: function ($sce) {
```

Comment:
Can we add an example of adding HTML to the message?

Reply:
How about something more like "Default <strong>notification</strong> message." ? I think we need to make it obvious so that folks don't think the toast message is always italicized.

Reply:
Didn't catch your response earlier, gotcha... could also place a CSS class for color in there and maybe limit the italics to a word instead of the whole message